### DISCUSSION TITLE
Capture prompt ordering

### DISCUSSION THREAD

**Discussion by bayradberry** [chhoumann/quickadd#331] [Q&A]
On Capture, when capturing task name and due date, can I make it ask for the task name first instead of the date?

**Comment by chhoumann** [comment]
The order is fixed, unfortunately. You could create a macro and ask in a preceding step, though.

**Comment by chhoumann** [comment]
Coming back to update this one, since my old answer is out of date.

Yes, this works now. The plain `{{VALUE}}` prompt (your task name) always comes before `{{VDATE}}` date prompts, so a capture format like this asks for the task first, then the date:

```
- [ ] {{VALUE}} 📅 {{VDATE:due,YYYY-MM-DD}}
```

Named `{{VALUE:...}}` prompts follow the order the tokens appear in the format string.

Even nicer: there is now a One-page inputs mode (Settings -> "One-page input for choices", currently Beta) that collects all of a choice's inputs in a single form, so prompt order mostly stops mattering - the task and date fields sit in one modal and you fill them in any order. One caveat: not every prompt is reorderable; for example, when a capture needs to ask which file to capture to, that picker still comes before the content prompts.

Docs: https://quickadd.obsidian.guide/docs/Advanced/onePageInputs
